contracts/MarketOracle.sol

```diff
@@ -29,12 +33,11 @@ contract MarketOracle is Ownable {
      *      Expired market sources are ignored. If there has been no trade volume in the last
      *      24hrs, then there is effectively no exchange rate and that value should be ignored by
      *      the client.
-     * @return exchangeRate: Volume weighted average of exchange rates.
-     *         volume: Total trade volume of the last reported 24 hours in Token volume.
+     * @return Volume weighted average of exchange rates.
      */
-    function getPriceAnd24HourVolume()
+    function getAggregatedData()
         external
-        returns (uint256, uint256)
+        returns (uint256)
```

Comment:
We lost the second parameter, which does carry information even if we don't use the "volume" part of that info in the policy.

What if we just changed this to a bool representing "valid" / "invalid" ? That would allow us to execute as before without failing, even when all the sources have expired.

Reply:
The generic version of the oracle should only return one type of data. If we want volume information in the policy, we would have another instance of the oracle on the policy to provide volume data.

I like the idea of adding the `validity` metric to the generic interface 👍.